src/lib/swapper/swappers/ThorchainSwapper/getThorTradeQuote/getTradeQuote.ts

```diff
@@ -137,375 +113,22 @@ export const getThorTradeQuote = async (
-            }
-          },
-        ),
-      )
-
-      const routes = maybeRoutes.filter(isFulfilled).map(maybeRoute => maybeRoute.value)
-
-      // if no routes succeeded, return failure from swapper
-      if (!routes.length)
-        return Err(
-          makeSwapErrorRight({
-            message: 'Unable to create any routes',
-            code: SwapErrorType.TRADE_QUOTE_FAILED,
-            cause: maybeRoutes.filter(isRejected).map(maybeRoute => maybeRoute.reason),
-          }),
-        )
-
-      // otherwise, return all that succeeded
-      return Ok(routes)
-    }
-
-    case CHAIN_NAMESPACE.CosmosSdk: {
-      const sellAdapter = assertGetCosmosSdkChainAdapter(sellAsset.chainId)
-      const feeData = await sellAdapter.getFeeData({})
-
-      return Ok(
-        perRouteValues.map(
-          ({
-            source,
-            quote,
-            expectedAmountOutThorBaseUnit,
-            isStreaming,
-            estimatedExecutionTimeMs,
-            affiliateBps,
-          }): ThorTradeQuote => {
-            const rate = getRouteRate(expectedAmountOutThorBaseUnit)
-            const buyAmountBeforeFeesCryptoBaseUnit = getRouteBuyAmount(quote)
-
-            const buyAmountAfterFeesCryptoBaseUnit = convertPrecision({
-              value: expectedAmountOutThorBaseUnit,
-              inputExponent: THORCHAIN_FIXED_PRECISION,
-              outputExponent: buyAsset.precision,
-            }).toFixed()
-
-            const updatedMemo = addSlippageToMemo({
-              affiliateFeesThorBaseUnit: quote.fees.affiliate,
-              expectedAmountOutThorBaseUnit,
-              quotedMemo: quote.memo,
-              slippageBps: inputSlippageBps,
-              isStreaming,
-              chainId: sellAsset.chainId,
-              affiliateBps,
-              streamingInterval,
-            })
-
-            return {
-              id: uuid(),
-              memo: updatedMemo,
-              receiveAddress,
-              affiliateBps,
-              isStreaming,
-              rate,
-              steps: [
-                {
-                  estimatedExecutionTimeMs,
-                  rate,
-                  sellAmountIncludingProtocolFeesCryptoBaseUnit: sellAmountCryptoBaseUnit,
-                  buyAmountBeforeFeesCryptoBaseUnit,
-                  buyAmountAfterFeesCryptoBaseUnit,
-                  source,
-                  buyAsset,
-                  sellAsset,
-                  accountNumber,
-                  allowanceContract: '0x0', // not applicable to cosmos
-                  feeData: {
-                    networkFeeCryptoBaseUnit: feeData.fast.txFee,
-                    protocolFees: getProtocolFees(quote),
-                    chainSpecific: {
-                      estimatedGasCryptoBaseUnit: feeData.fast.chainSpecific.gasLimit,
-                    },
-                  },
-                },
-              ],
-            }
-          },
-        ),
-      )
-    }
-
+  switch (tradeType) {
```

Comment:
This will always be `L1ToL1` when the feature flag is off, so there should be no behavioural change.